refinery/data_set_manager/utils.py

```diff
@@ -722,6 +730,21 @@ def generate_solr_params(params, assay_uuids, facets_from_config=False):
     return encoded_solr_params
 
 
+def cull_attributes_from_list(attribute_list, remove_facets):
+    """Helper method which will remove the first matching attribute from the
+    AttributeOrder based on the solr_field name.
+    Keyword Argument
+        attribute_list -- AttributeOrder list
+        remove_facets -- list of solr_field names"""
+    culled_attributes = attribute_list[:]
```

Comment:
Why the intermediate assignment?

If its just for clarity could we get rid of the `[:]`?